style.css

```diff
@@ -1,340 +1,1391 @@
+
+.blog__wrapper {
+    display: grid;
+    grid-template-columns: 1fr 1fr;
+    gap: 30px; 
+    margin-bottom: 30px;
+}
+
+.blog__post {
+    padding: 30px;
+    background-color: #1D1C28;
+}
+
+.blog__post--big {
+    grid-row: span 2;
+}
+
+.blog__image {
+    position: relative;
+}
+
+.blog__image img {
+    width: 100%;
+    height: 175px;
+    object-fit: cover;
+}
+
+.blog__post--big > .blog__image img {
+    height: 640px;
+}
+
+.blog__info {
+    display: flex;
+}
+
+.blog__name,
+.blog__date {
+    position: relative;
+    padding-left: 25px;
+}
+
+.blog__name {
+    padding-right: 36px;
+}
+
+.blog__name::before,
+.blog__date::before {
+    position: absolute;
+    content: '';
+    background-image: url('images/user.svg');
+    background-size: cover;
+    background-repeat: no-repeat;
+    top: 5px;
+    left: 0;
+    width: 16px;
+    height: 16px;
+}
+
+.blog__date::before {
+    background-image: url('images/calendar.svg');
+}
+
+.blog__name:after {
+    content: '';
+    position: absolute;
+    top: 4px;
+    right: 18px;
+    width: 2px;
+    height: 16px;
+    background-color: #D0D7FB2B;
+}
+
+/* Index page Our Team block */
+
+.team {
+    max-width: 1300px;
+    margin-inline: auto;
+}
+
+.team__members {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    gap: 30px;
+    margin-bottom: 100px;
+}
+
+.team__item {
+    padding: 32px;
+    background-color: #1D1C28;
+    text-align: center;
+}
+
+.team__image {
+    display: block;
+    border-radius: 50%;
+    margin-inline: auto;
+}
+
+/* Index page About block */
+
+.about__wrapper {
+    max-width: 960px;
+    margin: 0 auto;
+}
+
+.about__title {
+    position: relative;
+    margin-top: 55px;
+    font-family: var(--font-secondary);
+    font-size: 24px;
+    line-height: 32px;
+    color: #fff;
+    z-index: 2;
+}
+
+.about__text {
+    font-size: 18px;
+    line-height: 28px;
+    color: #fff;
+}
+
+.about__features {
+    display: flex;
+    flex-wrap: wrap;
+    margin-bottom: 30px;
+    gap: 30px;
+    counter-reset: tidbit-counter;
+    list-style-type: decimal-leading-zero;
+}
+
+.about__features li:before {
+    position: absolute;
+    top: 27px;
+    left: 30px;
+    content: "0" counter(tidbit-counter);
+    counter-increment: tidbit-counter;
+    -webkit-text-stroke: 2px var(--color-blue);
+    -webkit-text-fill-color: transparent;
+    font-family: var(--font-secondary);
+    font-size: 80px;
+    line-height: 96px;
+}
+
+.about__item {
+    flex-grow: 1;
+    position: relative;
+    width: 46%;
+    background-color: #1A1925;
+    box-sizing: border-box;
+    list-style: none;
+}
+
+.about__item a {
+    display: block;
+    padding: 30px;
+}
+
+.about__item a:hover {
+    background-image: url(images/hover-up-left.webp), url(images/hover-down-right.webp);            
+    background-repeat: no-repeat, no-repeat, no-repeat;
+    background-position: top left, bottom right;
+    transition: all 0.5s ease;
```

Comment:
transition should be on the element, not on the state. Please move to .about__item a

Reply:
done